flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/search/config/memory_managers/ballerina_memory.json

```diff
@@ -1,6 +1,6 @@
 {
   "description": "Ballerina memory managers search flow model",
-  "kind": "MEMORY_MANAGER",
+  "kind": "MEMORY",
```

Comment:
Why is the kind for memory managers are changed from `MEMORY_MANAGER` to `MEMORY` in here? Is it intentional?